### PR TITLE
chore(ru): remove `{{IncludeSubnav}}` macro

### LINKED ISSUE #5170: Remove deprecated {{IncludeSubnav}} macro
The `{{IncludeSubnav}}` macro has been [deprecated in February 2021](https://github.com/mdn/yari/blob/1c28a240f988803c175e62d3c28acc5cc055a7a5/kumascript/macros/IncludeSubnav.ejs) and does no longer produce any content since then.

Afaik no replacement exists, so occurrences can just be deleted.

### PR DESCRIPTION
**Pull request by yin1999**
### Description

chore: remove {{IncludeSubnav}} macro from ru. This macro is a [no-op](https://github.com/mdn/yari/blob/main/kumascript/macros/IncludeSubnav.ejs) macro, so we can just remove this.

And I've added some sidebar macro for documents in `learn`.

### Related issues and pull requests

Fixes: #5170
Part of: #5603

### CODE REVIEW

File: files/ru/games/tutorials/2d_breakout_game_pure_javascript/index.md [hunk 1]
```diff
@@ -11,9 +11,7 @@ tags:
   - Руководство
 translation_of: Games/Tutorials/2D_Breakout_game_pure_JavaScript
 ---
-{{GamesSidebar}}{{IncludeSubnav("/en-US/docs/Games")}}
-
-{{Next("Games/Tutorials/2D_Breakout_game_pure_JavaScript/Создание_Canvas_и_рисование_на_нём")}}
+{{GamesSidebar}}{{Next("Games/Tutorials/2D_Breakout_game_pure_JavaScript/Создание_Canvas_и_рисование_на_нём")}}
```
Comment (yin1999) [inline files/ru/games/tutorials/2d_breakout_game_pure_javascript/index.md:14]:
```suggestion

{{GamesSidebar}}{{Next("Games/Tutorials/2D_Breakout_game_pure_JavaScript/Создание_Canvas_и_рисование_на_нём")}}
```

File: files/ru/games/tutorials/index.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ tags:
   - уроки
 translation_of: Games/Tutorials
 ---
-{{GamesSidebar}}{{IncludeSubnav("/en-US/docs/Games")}}
+{{GamesSidebar}}
```
Comment (yin1999) [inline files/ru/games/tutorials/index.md:17]:
```suggestion

{{GamesSidebar}}
```

**Review by sashasushko — APPROVED** [review]
LGTM 🔥

**Comment by sashasushko** [comment]
@yin1999 thanks a lot. I resolved conflicts and merged

**Pull request merged**